grammars.yml

```diff
@@ -1361,9 +1358,16 @@ vendor/grammars/xmake-lua.tmbundle:
 vendor/grammars/xml.tmbundle:
 - text.xml
 - text.xml.xsl
+vendor/grammars/bru.tmLanguage.json:
+- source.bru
 vendor/grammars/zeek-sublime:
 - source.zeek
 vendor/grammars/zenstack:
 - source.zmodel
 vendor/grammars/zephir-sublime:
 - source.php.zephir
+
+
+vendor/grammars/vscode-ruby-slim:
+- text.slim
+
```

Comment:
🤔 looks like this file has been manually edited. Don't. It's kept up to date by various scripts that are run when you follow the instructions in the CONTRIBUTING.md file. The only change we should see in this file is the addition of your new grammar.